### DISCUSSION TITLE
How to make macos browser thing?

### DISCUSSION THREAD

**Discussion by JojoplayzMC** [zauberzeug/nicegui#2066] [Q&A]
On your website, it said you made it using nicegui, but how did you get those macos browser tab thingy?
![image](https://github.com/zauberzeug/nicegui/assets/129040463/f532b585-8279-44ee-8472-7f3da46bd6db)

**Comment by falkoschindler** [comment]
It's all hand-made with Tailwind and CSS:
https://github.com/zauberzeug/nicegui/blob/c0bf52f294696f7112324b2436e5251a3a669a08/website/documentation/demo.py#L75-L111

**Reply by JojoplayzMC** [reply]
Thanks :)
